testgrid/cmd/configurator/config_test.go

```diff
@@ -52,7 +53,13 @@ var (
 		"sig",
 		"wg",
 	}
-	prefixes = [][]string{orgs, companies}
+	dashboardPrefixes = [][]string{orgs, companies}
+
+	// gcs prefixes populated by the kubernetes prow instance
+	prowGcsPrefixes = []string{
```

Comment:
Maybe even a non-gcs-specific `prowArtefactPathPrefixes` kind of thing (assuming `prowGcsPrefixes` is the path where the build/test artefacts are stored and I haven't got this all wrong 😅 ). `prowGcsPrefixes` is already descriptive enough though.

Reply:
in this case I was sticking with gcs prefix because that's the name of the field in the config proto / yaml